qa/acceptance/spec/shared_examples/cli/logstash-plugin/generate.rb

```diff
@@ -13,18 +13,19 @@
   end
 
   describe "on #{logstash.hostname}" do
-
```

Comment:
all changes in this generate.rb file are whitespace related, can we take them out of this PR?